langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaClient.java

```diff
@@ -200,6 +209,23 @@ public OllamaModelCard showInformation(ShowModelInformationRequest showInformati
         }
     }
 
+    private Interceptor headerInterceptor(Map<String, String> customHeaders) {
+        return new Interceptor() {
+            @NotNull
+            @Override
+            public okhttp3.Response intercept(@NotNull Chain chain) throws IOException {
+                Request original = chain.request();
+                Request.Builder requestBuilder = original.newBuilder()
+                        .method(original.method(), original.body());
```

Comment:
is there a reason to re-define method and body? [GenericHeaderInjector](https://github.com/ai-for-java/openai4j/blob/e4d41a0463c6ad8fdd88d709026dbab2a762fe67/src/main/java/dev/ai4j/openai4j/GenericHeaderInjector.java#L22) seems to work fine without it.

Reply:
Actually no reason...already removed.